src/core/manager.h

```diff
@@ -546,15 +527,14 @@ int manager_add_job_full(
                 Set *affected_jobs,
                 sd_bus_error *error,
                 Job **ret);
-static inline int manager_add_job(
+int manager_add_job(
                 Manager *m,
                 JobType type,
                 Unit *unit,
                 JobMode mode,
                 sd_bus_error *error,
-                Job **ret) {
-        return manager_add_job_full(m, type, unit, mode, 0, NULL, error, ret);
-}
```

Comment:
also here

Reply:
This requires pulling in transaction.h into the header so not here